docs/NativeComponentsIOS.md

```diff
@@ -229,68 +242,75 @@ var RCTSwitch = requireNativeComponent('RCTSwitch', Switch, {
 
 ## Events
 
-So now we have a native map component that we can control easily from JS, but how do we deal with events from the user, like pinch-zooms or panning to change the visible region?  The key is to declare an event handler property on `RNTMapManager`, make it a delegate for all the views it vends, and forward events to JS by calling the event handler block from the native view.  This looks like so (simplified from the full implementation):
+So now we have a native map component that we can control easily from JS, but how do we deal with events from the user, like pinch-zooms or panning to change the visible region?
+
+Until now we've just returned a `MKMapView` instance from our manager's `-(UIView *)view` method. We can't add new properties to `MKMapView` so we have to create a new subclass from `MKMapView` which we use for our View. We can then add a `onRegionChange` callback on this subclass: 
 
 ```objectivec
-// RNTMap.h
+// RNTMapView.h
 
 #import <MapKit/MapKit.h>
 
 #import <React/RCTComponent.h>
 
-@interface RNTMap: MKMapView
+@interface RNTMapView: MKMapView
 
-@property (nonatomic, copy) RCTBubblingEventBlock onChange;
+@property (nonatomic, copy) RCTBubblingEventBlock onRegionChange;
 
 @end
-```
 
-```objectivec
-// RNTMap.m
+// RNTMapView.m
 
-#import "RNTMap.h"
+#import "RNTMapView.h"
 
-@implementation RNTMap
+@implementation RNTMapView
 
 @end
 ```
 
-```objectivec
-// RNTMapManager.m
+Next declare an event handler property on `RNTMapManager`, make it a delegate for all the views it exposes, and forward events to JS by calling the event handler block from the native view.
 
-#import "RNTMapManager.h"
+```objectivec{9,17,31-48}
```

Comment:
Does `objectivec{9,17,31-48}` creates a highlight for the ReactNative docs as well?